RadixWallet/Features/TransactionReviewFeature/TransactionReview.swift

```diff
@@ -721,17 +699,17 @@ extension TransactionReview {
 		var manifest = reviewedTransaction.transactionManifest
 		if case let .success(feePayerAccount) = reviewedTransaction.feePayer.unwrap()?.account {
 			do {
-				manifest = try reviewedTransaction.transactionManifest.withLockFeeCallMethodAdded(
-					address: feePayerAccount.address.asGeneral,
-					fee: reviewedTransaction.transactionFee.totalFee.lockFee
+				manifest = try reviewedTransaction.transactionManifest.modify(
```

Comment:
Is this using the modify from RET? I recall that we didn't want to use it because it is extract smart, and would add `lock_fee_and_withdraw` than just `lock_fee` instruction, but maybe it is not the case anymore.

Reply:
I'm using Scrypto's transaction builder, and it does:

```rust
    InstructionV1::CallMethod {
            address: address.into(),
            method_name: "lock_fee",
            <AMOUNT>
     }
```

do you think it is wrong?

Reply:
Based on testing I think this is correct...

Reply:
That is why I was asking if you are using the RET manifest modify. And I see that you are not and the behaviour is as expected 👍.